spec/spec_helper.rb

```diff
@@ -1,10 +1,14 @@
+# Define all environment varibles for specs
+ENV['RABBIT_EVENT_BUS_APP_NAME'] = "app_name"
+ENV['RABBIT_EVENT_BUS_TOPIC_NAME'] = "event_bus"
+ENV['RABBIT_EVENT_BUS_VHOST'] = "event_bus"
+ENV['RABBIT_URL'] = "amqp://guest:guest@localhost:5672"
```

Comment:
wdyt about using `ENV['RABBIT_URL'] ||= "amqp://guest:guest@localhost:5672"`? That way it is easy to overwrite it if we want to.

Reply:
Don't worry with that, the reason I moved theses guys to spec_helper was to remove the dependency with `dotenv` gem and drop unnecessary configurations on development environment 🙂